book/src/jsonrpc-api.md

```diff
@@ -27,6 +27,7 @@ Methods
 * [getLastId](#getlastid)
 * [getSignatureStatus](#getsignaturestatus)
 * [getTransactionCount](#gettransactioncount)
+* [getConfirmation](#getconfirmation)
```

Comment:
How about `getConfirmationTime`? I find `getConfirmation` to be a little ambiguous.
Also, let's alphabetize this list?

Reply:
Maybe even getMeanConfirmationTime to distinguish it from furture API that returns the confirmation time of a particular block?

Reply:
It returns the confirmation time of the latest block, the current API is not that smart: https://github.com/solana-labs/solana/blob/16877ecf037ae2b41cb2ceaab1fb5048a61f9d9c/src/compute_leader_confirmation_service.rs#L75